package/gluon-client-bridge/files/lib/gluon/upgrade/320-gluon-client-bridge-wireless

```diff
@@ -21,14 +21,20 @@ local function configure_client(config, radio, index, suffix)
 
   uci:delete('wireless', name)
 
+  if index == 0 then
+    macaddr = util.generate_mac(0)
+  elseif index == 1 then
+    macaddr = util.generate_mac(3)
+  end
+
```

Comment:
I also thought about this... I decided to do it this way, because it could lead to missunderstandings. Somebody could try to generate a mac for the interface _ibss4_ and the collision could be undetected for a while...

Maybe something like this?:
``` lua
if index < 3 then
  macaddr = util.generate_mac(index * 3)
end
```

I think an empty mac is more conspicuous than a colliding..